<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPlantWindow</class>
 <widget class="QWidget" name="AddPlantWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="plantTypeBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>181</width>
     <height>147</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Attribute(s)</string>
   </property>
   <widget class="QCheckBox" name="flowering">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Flowering</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="succulent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Succulent</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="tropical">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Tropical</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="foliage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Foliage</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fern">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Fern</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="edible">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Edible</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="aquatic">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Aquatic</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="airplant">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Air Plant</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="tree">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Tree</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="petsafe">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Pet-Safe</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="conifer">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Conifer</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cactus">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Cactus</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Additional Notes</string>
   </property>
   <widget class="QTextEdit" name="notesEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="plantInfoBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Basics</string>
   </property>
   <widget class="QTextEdit" name="plantNameEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Plant Name</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateObtainedEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date Obtained</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="addPlantToActive">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Plant!</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Water/Fertilizer</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Watered</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="lastWateredDate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Water every</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="waterDays">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>days!</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
